src/components/templates/layout/index.native.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import styled from '@emotion/native';
```

Comment:
All the emotion references are going to need refactoring as Peter moved us back to styled-components

Reply:
@skywickenden I've just rebased in those PRs, updated and force-pushed.